src/Integration/LocalServices/IProjectSystemHelper.cs

```diff
@@ -77,14 +77,6 @@ public interface IProjectSystemHelper : ILocalService, IVsHierarchyLocator
         /// </summary>
         IEnumerable<Project> GetSolutionProjects();
 
-        /// <summary>
-        /// Returns only the filtered project based on a common filter.
-        /// This should only be called after we connected to a SonarQube server,
-        /// since some of the filtering is SonarQube server instance specific.
-        /// <seealso cref="IProjectSystemFilter"/> which is used internally.
-        /// </summary>
-        IEnumerable<Project> GetFilteredSolutionProjects();
```

Comment:
This was previously used by the binding code, but I removed that call during earlier cleanup. Nothing else uses this method now, so I've deleted it from the interface.

That in turn means the IProjectSystemFilter is no longer used. I'll remove that in the next PR.